content/en/docs/getting-started/hardware-requirements.md

```diff
@@ -5,49 +5,47 @@ description: "Define the hardware requirements for your Cozystack use case."
+The good news is that whichever service you need, Cozystack will run it prefectly: securely, efficiently, and
+in a fully containerized or virtualized environment.
 
 Hardware requirements depend on your usage scenario.
-Below are several common deployment options—review them to determine what setup best fits your needs.
+Below are several common deployment options, review them to determine which setup fits best to your needs.
 
-## Proof-of-Concept Lab
+## Small lab
 
-Here are the baseline requirements for running a proof-of-concept installation.
+Here are the baseline requirements for running a tiny installation.
 
 **Compute:**
 
-- Three physical or virtual servers with amd64/x86_64 architecture, each having at least 16 GB RAM and 4 CPU cores.
-- If the servers are virtualized, ensure that nested virtualization is enabled and the CPU model is set to `host` (without emulation).
-- A PXE installation requires an extra management VM or physical server connected to the same network,
-  with any Linux system installed on it (for example, Ubuntu should be enough).
-  This VM should support the `x86-64-v2` architecture, which most probably can be achieved by setting CPU model to `host`.
+- Three physical or virtual servers with amd64/x86_64 architecture, with at least 16 GB RAM and 4 CPU cores each.
+- Virtualized servers need nested virtualization enabled and the CPU model set to `host` (without emulation).
+- PXE installation requires an extra management instance connected to the same network, with any Linux system able to run a Docker container.
+  It should also have `x86-64-v2` architecture, which most probably may be achieved by setting CPU model to `host` in case of a VM.
 
-Storage:
+**Storage:**
 
-- The primary disk should be at least 32 GB. Talos Linux uses it for the OS, images, and logs. It's better to have more space.  
-- A secondary disk is required for the replicated storage system. 100 GB is a good starting point, but the actual size depends entirely on your workload.
+- Primary disk should be at least 32 GB. Talos Linux uses it for the OS, images, and logs. It would be nice to have more.
+- Secondary disk is dedicated to the storage system. 100 GB is a good starting point, but the actual size depends entirely on your workload.
 
 **Networking:**
 
-- Machines must be allowed to use additional IPs (this is typically disabled by default and must be enabled explicitly in most public clouds), 
-  or an external load balancer must be available.
-- Additional public IPs for ingress and virtual machines may be needed. Not all public cloud providers support floating IPs.
+- Machines should be able to use additional IPs, or an external load balancer should be available.
+  In public clouds the use of the additional IPs typically needs to be allowed explicitly.
+- Additional public IPs for ingress and virtual machines may be needed. Check if your public cloud provider supports floating IPs.
```

Comment:
```suggestion
- Machines must be allowed to use additional IPs, or an external load balancer must be available.
  Using additional IPs is disabled by default and must be enabled explicitly in most public clouds.
- Additional public IPs for ingress and virtual machines may be needed. Check if your public cloud provider supports floating IPs.
```

Reply:
There's this important context about the option being available but disabled by default, which I don't want to lose.